cmd/kube-apiserver/app/server.go

```diff
@@ -569,7 +575,48 @@ func BuildStorageFactory(s *options.ServerRunOptions) (*serverstorage.DefaultSto
 	}
 
 	if s.Etcd.EncryptionProviderConfigFilepath != "" {
-		transformerOverrides, err := encryptionconfig.GetTransformerOverrides(s.Etcd.EncryptionProviderConfigFilepath)
+		// Obtain GCE cloud for a Google Cloud KMS connection if needed.
+		// cloudproviders are not yet in the staging directory of Kubernetes, and thus we cannot access
+		// the gce package (sub-package of cloudproviders) from inside the staging directory.
+		// We need to be able to create a gce cloud object inside the staging directory. Thus, for now,
+		// we pass a function which does that, from here.
+		// TODO: If and when cloudproviders is migrated to the staging directory, move the logic of this
```

Comment:
It may or may not. I'll make it say "if cloudproviders is migrated from the current location" and remove the TODO perhaps.